### PR TITLE
pep8, callback, small bugfix, spelling

### PR DESCRIPTION
**Pull request by kevinkk525**
bugfix in esp_link.esp_link.
Implemented server_status() as callback as suggested (hope I understood your comment correctly)
Spelling corrections in README.

### CODE REVIEW

File: esp_link/esp_link.py [hunk 1]
```diff
@@ -115,16 +116,10 @@ def __init__(self, loop, verbose):
...
-        oldstatus = True
-        while True:
-            await asyncio.sleep_ms(500)
-            status = self.cl.status()
-            if status != oldstatus:
-                oldstatus = status
-                data = ['status', status]
-                line = ''.join((ujson.dumps(data), '\n'))
-                await self.swriter.awrite(line)
+    async def server_status(self, status):
```
Comment (peterhinch) [inline esp_link/esp_link.py]: That is what I had in mind. Have you tested it?

File: esp_link/esp_link.py [hunk 2]
```diff
@@ -52,14 +53,14 @@ def __init__(self, loop, verbose):
         self.qos = config[6]
         # Handle case where ESP8266 has not been initialised to the WLAN
         sta_if = network.WLAN(network.STA_IF)
-        ap = network.WLAN(network.AP_IF) # access-point interface.
-        ap.active(False)         # deactivate AP interface.
+        ap = network.WLAN(network.AP_IF)  # access-point interface.
```
Comment (peterhinch) [inline esp_link/esp_link.py]: Thanks for this, but I have substantially changed this in that I subclass Client and implement bad_wifi. So most of this code is gone.

I suggest holding off any further changes to esp_link.py until I commit a stable version.

File: pb_link/README.md [hunk 3]
```diff
@@ -207,7 +207,7 @@ application, ensuring that the Pyboard app communicates with its matching
 server app. ID's must be newline-terminated and must not include other newlines.
 
 Subclasses must define a synchronous `start` bound method. This takes no args.
-Typically it lunches user coroutines.
+Typically it launches user coroutines.
```
Comment (peterhinch) [inline pb_link/README.md]: What a howler :-))

File: esp_link/esp_link.py (hunk 1 above)
Comment (kevinkk525) [reply esp_link/esp_link.py]: No I have not as I don't have any pyboard. It is implemented in the same way as my examples, so it should work but I'd have configure another esp to actually read the message that the coroutine sends using i2c.

Comment (peterhinch) [reply esp_link/esp_link.py]: OK, no problem. It looks good, and I'll test when I merge.

**Comment by peterhinch** [comment]
A point of code style. In the Client constructor we have:
```python
  connected_cb=None, connected_cb_args=None,

        self._concbargs = () if connected_cb_args is None else connected_cb_args
```
I used this formulation for consistency with your use of `None` in the other args.
My usual approach is:
```python
# global namespace
eliza = lambda *_ : None
# Constructor args
   connected_cb=eliza, connected_cb_args=(),
```
This avoids the need for any conditionals in subsequent code.

I'm happy to go with either. What's your preference?

**Comment by kevinkk525** [comment]
I just tried to find some standard way of doing this but did not find anything.
I prefer to have it explicitly None as it is more obvious but we can go with your style too.

Although *eliza* would remove the need for a check if *connected_cb* is None, it would include calling a global function *launch* which makes it about 200us slower when trying in repl than the check if *connected_cb* is None (in repl context also checking a global variable).

**Comment by peterhinch** [comment]
OK, I'm convinced. I'll leave it as it is.

**Pull request merged**